packages/easy-coding-standard/scoper.php

```diff
@@ -110,6 +110,15 @@ function (string $filePath, string $prefix, string $content): string {
             return $content;
         },
 
+        // fixes https://github.com/symplify/symplify/issues/3205
+        function (string $filePath, string $prefix, string $content): string {
+            if (! Strings::endsWith($filePath, 'src/Testing/AbstractKernelTestCase.php')) {
+                return $content;
+            }
+
+            return Strings::replace($content, '#use ' . $prefix . '\\\\PHPUnit\\\\Framework\\\\TestCase#', 'use PHPUnit\Framework\TestCase');
```

Comment:
you can remove `use ` to ensure it works on fqcn with \ prefix

Reply:
Done in 38fd92ccb1a1e9152bda38257358801940f0af64.